jwt/utils.py

```diff
@@ -133,24 +133,26 @@ def is_pem_format(key: bytes) -> bool:
 # Based on https://github.com/pyca/cryptography/blob/bcb70852d577b3f490f015378c75cba74986297b/src/cryptography/hazmat/primitives/serialization/ssh.py#L40-L46
 _CERT_SUFFIX = b"-cert-v01@openssh.com"
 _SSH_PUBKEY_RC = re.compile(rb"\A(\S+)[ \t]+(\S+)")
-_SSH_KEY_FORMATS = [
+_SSH_KEY_FORMATS = (
     b"ssh-ed25519",
     b"ssh-rsa",
     b"ssh-dss",
     b"ecdsa-sha2-nistp256",
     b"ecdsa-sha2-nistp384",
     b"ecdsa-sha2-nistp521",
-]
+)
 
 
 def is_ssh_key(key: bytes) -> bool:
-    if any(string_value in key for string_value in _SSH_KEY_FORMATS):
+    if key.startswith(_SSH_KEY_FORMATS):
         return True
 
-    ssh_pubkey_match = _SSH_PUBKEY_RC.match(key)
-    if ssh_pubkey_match:
-        key_type = ssh_pubkey_match.group(1)
-        if _CERT_SUFFIX == key_type[-len(_CERT_SUFFIX) :]:
-            return True
+    # Avoid the regex if the _CERT_SUFFIX is not in the key
+    if _CERT_SUFFIX in key:
+        ssh_pubkey_match = _SSH_PUBKEY_RC.match(key)
+        if ssh_pubkey_match:
+            key_type = ssh_pubkey_match.group(1)
+            if _CERT_SUFFIX == key_type[-len(_CERT_SUFFIX) :]:
+                return True
```

Comment:
I'm not sure this is even needed since all the examples I can find always start with `_SSH_KEY_FORMATS`

I think the entire function might be able to be simplified down to 

return  `key.startswith(_SSH_KEY_FORMATS)`

Reply:
Looks like startswith is enough
```
src/cryptography/hazmat/primitives/serialization/ssh.py:_CERT_SUFFIX = b"-cert-v01@openssh.com"
tests/hazmat/primitives/test_ssh.py:            b"ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbm"
tests/hazmat/primitives/test_ssh.py:            load_ssh_public_identity(b"ssh-rsa-cert-v01@openssh.com invalid")
tests/hazmat/primitives/test_ssh.py:                b"ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAK0VjZHNhLXNoYTI"
tests/hazmat/primitives/test_ssh.py:            b"ssh-rsa-cert-v01@openssh.com AAAAHHNzaC1yc2EtY2VydC12MDFAb3Blbn"
tests/hazmat/primitives/test_ssh.py:            b"ssh-ed25519-cert-v01@openssh.com AAAAIHNzaC1lZDI1NTE5LWNlcnQtdj"
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/dsa-p256.pub:ssh-dss-cert-v01@openssh.com AAAAHHNzaC1kc3MtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgIE0DERxEocGf8SILUWuT5wakv34qIiz0+1/fQVf0PYQAAACBAPloKFUpz5YJF4doiftzBT3wlM37PvuklYVczyOr8HCn4srUK4Zwe13Ce/Ee+Ibpy3nECDI9AICjYycWOVVbpDAy4nFDpsWal/nxL1wFIocSVD3eiDK0sCJI0JoKN8CtARnX4EmMwa8VJW9VyHZyRQ2LMKHsIY3WYoJswmMk+uRtAAAAFQCbeL/Q/whJsTM5Mz42+p2NA0MAJwAAAIEAjhE+6yBllAn2AFUoMN9kC5dSxWrcVG8flczdhGDfryTTO7ouUlZEQXjtgc8XDaz3ABttq8WDmihsJ10F9QY/jh4D6ml8tFL4F59CiiImZHSnUv5GBVzkp/P26a9q/PJr9VwQEhIwDQ4lEOHPApBTQFMjDhEO+/5seqWDerzcuJkAAACAJn80y6FFao0jV4WVK8O5x2DFAmX2ic3DcjZiehRtB7ar9FQJfkLRFgq4AqR683n2DyO2Mta4mNVuLEcmHO+uC08BRbwx3CuftAXUDsMK+4bcZFQGNtBfIQHugA58sIvjBENlEfbZwNNLiyzKtG7W+61WZEnNb5FBNHH8lSCMJ04AAAAAAAAAAAAAAAEAAAAEdGVzdAAAABAAAAADZXZlAAAABWFsaWNlAAAAAGS0cnQAAAAAY8XhBAAAACAAAAAYY3JpdGljYWxAY3J5cHRvZ3JhcGh5LmlvAAAAAAAAAAAAAAAAAAAAaAAAABNlY2RzYS1zaGEyLW5pc3RwMjU2AAAACG5pc3RwMjU2AAAAQQSVuxZ77Yb35R81SnzVzIhFlOJ8KDTC7O3KL/OFzutJJ71Uah4j9ixchyr+uY3SpUnE0NqHqKTZYOMe/9MJzCdWAAAAZAAAABNlY2RzYS1zaGEyLW5pc3RwMjU2AAAASQAAACEA99MMfmfSFIYvB1EF3s7mWXqUsch4sdf+I28jr1SUKD8AAAAgY04RTcT1vqMzs5bVoO0vF/RfQyr06Z1IDzCX1l3h70k=
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-dsa.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgqlL0IDRu6PBXcHrEqr66w+nW4TWpgWcqB9/SiSVsHk0AAAAIbmlzdHAyNTYAAABBBKxrpV0taLLnqbE08WQoMGmopbPBTC1Vj3GLi4/+n9BINlKBskAjhVfQdfLzse4REtYbNH2iOseHS/r1l+jj2REAAAAAAAAAAAAAAAEAAAAEdGVzdAAAABAAAAADZXZlAAAABWFsaWNlAAAAAGS0cnQAAAAAY8XhBAAAACAAAAAYY3JpdGljYWxAY3J5cHRvZ3JhcGh5LmlvAAAAAAAAAAAAAAAAAAABsgAAAAdzc2gtZHNzAAAAgQC2BuglMCTWhV809q9omijAGP+z54k1BwCdFngTXr4k4Nm5DAsbUUaaQuELVK+oeB/iClhq17ghSM6gbGRZRZYlyBNfp05w77aKzw7w+0Z3JY8FDjwvsE6hWK7oWTUIF+79Nwe6QdHuryVsXKOG6YmKC+CPzSnpZ8jEGtCnI/HGqQAAABUA0UVWDteeovroX1jW5a0kbBbqnxUAAACAYkibo7SpYozbSObn2YlwLPa0phfpYcLmtwtZFaraYcJWsUSA8F4UATXNCxVPqwsl7PJfmGcJ+T+wvL16VxoCXsGqOgSNox3eVkGlwxF964AQ8BON77gi+ho6zw9ALoHUEDAKLwupRKXLoqLoRe+0T3//zokR3PX1JMbdraIgBcUAAACBAK41XlUiNgUc2fgcRhcPgFP7Ru1NxQ/a+muyfbcWaHPrOW5Jb90MfhmIqktc6tbqUceGtH2qqch1lxdaCw4e1sjeDsd0wnrIQcg4occQqZ2PYu3K3tcclyBXtVbzpxXq18QPUtl9idIDZ17L64iZgO1+EKlMsl0dZBtH37SWxIoWAAAANwAAAAdzc2gtZHNzAAAAKIvmGjVssyGoO4sCGCyz2wDCXUtMMbDxAp8nKRk6H0XdXoixPGY0oAo=
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-ed25519-non-singular-crit-opt-val.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIbmlzdHAyNTYAAABBBCZWRs4GYIHGJpyXuqvfFGWN49dnJRkZJLDkFrHf6mNHhIMI3vtrLfCZwxPSfnCYWK6YofssZ1FYA6TkVJq8Xi8AAAAAAAAAAAAAAAEAAAAAAAAAAAAAAABjsM0AAAAAAGOyHoAAAABtAAAADWZvcmNlLWNvbW1hbmQAAABBAAAAKGVjaG8gYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWEAAAARaW52YWxpZF9taXNjX2RhdGEAAAAPdmVyaWZ5LXJlcXVpcmVkAAAAAAAAABIAAAAKcGVybWl0LXB0eQAAAAAAAAAAAAAAMwAAAAtzc2gtZWQyNTUxOQAAACDdZgztgAFFC7T5PifrUy/kMu0Pnwq1au3vStKHe7FFMAAAAFMAAAALc3NoLWVkMjU1MTkAAABAt/0pBSDBFy1crBPHOBoKFoxRjKd1tKVdOrD3QVgbBfpaHfxi4vrgYe6JfQ54+vu5P+8yrMyACekT8H6hhvxHDw==
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-ed25519-non-singular-ext-val.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIbmlzdHAyNTYAAABBBCZWRs4GYIHGJpyXuqvfFGWN49dnJRkZJLDkFrHf6mNHhIMI3vtrLfCZwxPSfnCYWK6YofssZ1FYA6TkVJq8Xi8AAAAAAAAAAAAAAAEAAAAAAAAAAAAAAABjsM0AAAAAAGOyHoAAAAAXAAAAD3ZlcmlmeS1yZXF1aXJlZAAAAAAAAAAvAAAAFGNvbnRhaW5zLWV4dHJhLXZhbHVlAAAAEwAAAAVoZWxsbwAAAAYgd29ybGQAAAAAAAAAMwAAAAtzc2gtZWQyNTUxOQAAACDdZgztgAFFC7T5PifrUy/kMu0Pnwq1au3vStKHe7FFMAAAAFMAAAALc3NoLWVkMjU1MTkAAABAY80oIEvooz/k3x9a+yVkjSNRfi4y/q87wVYiT7keTpP4n9JV/Vlc0u7O2QYOHfb4DUkcrvbsksKVsiqoQu5qDg==
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-p256-broken-signature-key-type.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAg5F1PDG/y2CzJp73derFFtoa/OJjJQZ1jaQKqf2nKciQAAAAIbmlzdHAyNTYAAABBBL4QG3yEahvircH4Px+aVHip40dqWot+JJB3TuyFGN5rCDP4eZmM5gmTIDJw+Y3uJk/8oP44nSXmUhceEI01LlMAAAAAAAAAAAAAAAEAAAAEdGVzdAAAABAAAAADZXZlAAAABWFsaWNlAAAAAGS0cnQAAAAAY8XhBAAAACIAAAAaaW52YWxpZHNpZ0BjcnlwdG9ncmFwaHkuaW8AAAAAAAAAAAAAAAAAAABoAAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBINdZJ7jyyOqECwnOcDNw8OHgHwTU8uexxAU8Qs/QYv3bmjR7ExP9QVGKzWJ2LxyCzIwt+wjjt/7Y/DGWJ5ja5kAAABeAAAADHJzYS1zaGEyLTI1NgAAAEoAAAAhAIYe8pC0okwX8m6lbsdSNZFhNl5GXGKMvy0Czi33NktHAAAAIQDLKohBedhNdftYor3+G2mSFAJFGM/fDJgs8L3cOEsYfQ==
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-p256-duplicate-crit-opts.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgLhqcBHfeQKtL85NO1EYeTF6EWUPuCswcuhsj4jN8+9gAAAAIbmlzdHAyNTYAAABBBMnYLqlS1l6aEf5MAzEnTd2rfYYAdaesNxUsSW+2/QyIE0KiR1OOqnth+V+8JDTWBf78EcqxLOaz7Icw8d4dOzAAAAAAAAAAAAAAAAEAAAAEdGVzdAAAABAAAAADZXZlAAAABWFsaWNlAAAAAGS0cnQAAAAAY8XhBAAAAEIAAAAZZHVwbGljYXRlQGNyeXB0b2dyYXBoeS5pbwAAAAAAAAAZZHVwbGljYXRlQGNyeXB0b2dyYXBoeS5pbwAAAAAAAAAAAAAAAAAAAGgAAAATZWNkc2Etc2hhMi1uaXN0cDI1NgAAAAhuaXN0cDI1NgAAAEEEbA70TLh7ri83WgjRWsQc0dP146vVEFhHqSdKHw8TMScFt4pcJnBcCPmr2TvxUdNcDHfum11bQWuAxvOOZ/RfxAAAAGQAAAATZWNkc2Etc2hhMi1uaXN0cDI1NgAAAEkAAAAhAOP5SYGetAQ849rV+ezStHi2WRkRLA6QnBrWXFWjtTNzAAAAIESwusuQ5ucNFnxR/B+mF8Fcwk22mx09i5vATfJSTNxW
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-p256-duplicate-extension.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgN6rRGEpLjA1BohPwr8SEa33Vn4+H+94Jifl6hg6n5IIAAAAIbmlzdHAyNTYAAABBBFQDtz1+0wnP2wLnHZhES+XJVxImzusYhXd3G2cWCZL4HKZHRMIWj1x6oU9JdoEOMHppb6DK/OIFsXHwJeRBOOwAAAAAAAAAAAAAAAEAAAAEdGVzdAAAABAAAAADZXZlAAAABWFsaWNlAAAAAGS0cnQAAAAAY8XhBAAAAAAAAAA+AAAAF3Blcm1pdC1hZ2VudC1mb3J3YXJkaW5nAAAAAAAAABdwZXJtaXQtYWdlbnQtZm9yd2FyZGluZwAAAAAAAAAAAAAAaAAAABNlY2RzYS1zaGEyLW5pc3RwMjU2AAAACG5pc3RwMjU2AAAAQQQlhngteNXbqpH+iOYQyIWciB0SCfHkjXir3gZXszfzWka1sP0LIHAtszMTofPG0+SosIKtZWfiJvh0BoxtbthJAAAAZQAAABNlY2RzYS1zaGEyLW5pc3RwMjU2AAAASgAAACEAyAyVc+9ORWbQ/LdtR82+rKUxfMkD1wWDarYHk4mAWWIAAAAhALnEsJAtB4AVsNIC8COVeEEjmsPSQt4seHF6V5u7IWw9
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-p256-empty-principals.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgBXWXwDzsP++5IxnXmRrXXXP5wyJzMXW1nnQh7DPYZx8AAAAIbmlzdHAyNTYAAABBBCuO1AQsw/HmbXrjAHBJ7YA1ydYJeNbZ6LSArnxvrTX9kXDWF4bfOTSpUsC6OBKfzOiryc2Jr7QzXsFaOX0KPXUAAAAAAAAAAAAAAAEAAAAEdGVzdAAAAAAAAAAAZLRydAAAAABjxeEEAAAAAAAAAAAAAAAAAAAAaAAAABNlY2RzYS1zaGEyLW5pc3RwMjU2AAAACG5pc3RwMjU2AAAAQQRKq69tU5OhiF+0NOuZQU7X9BPL9aiW9Srxh5nizieNn1TLRkdpUXJisOk0t9Q5OgeFmo2JOcY3Cc2zXpBKWmp0AAAAZQAAABNlY2RzYS1zaGEyLW5pc3RwMjU2AAAASgAAACEAyfMU07vNgYpI+vi8O3XmBkKnuYFcofPoaq0H1FCWj0oAAAAhAIhv9JCLdJM2fKc++EFu51glQUggTzgBQR+zR8HEbpFN
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-p256-invalid-cert-type.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgSgPwfPbsfbii+9RAUJYav6zJ0oU54htu+vOuTdctY88AAAAIbmlzdHAyNTYAAABBBCARilffhR67GJ6CjUyP78jhu8eGCJ6bb68l86BnJIudjlYAQ/sgiFsQjmrMmDVqNrKGFudmJ3l+Nr78qump4pcAAAAAAAAAAAAAADIAAAAAAAAAAAAAAABktHJ0AAAAAGPFmrQAAAAAAAAAAAAAAAAAAABoAAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBK8dtlOuaMxRf6l2vYPkXBalJrwPuvA95aXVn7Y4TEwfKwlztl9x2kYVPq0z1zCcrST5xbObEkqpvwI7ibEYVNkAAABkAAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAABJAAAAIGxkTfbvXgevWi7ggjKUhaXQwT8cAKFoBaR1MX2602CGAAAAIQDmhJ+Jb4ZEb1vJU/9ekCUkAJg4MJ2nxLO4Rh02Gd1kDg==
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-p256-non-lexical-crit-opts.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAghD4Bg5C3PR6Yjo0Onv9jTmMJL/ysWxOWM3lvbjkwSbgAAAAIbmlzdHAyNTYAAABBBMXF33sjujlcCsz5pnR7SXXO34s49Ofqw6wIoqNe2zJy2MxaBTQzMi+WXLTpv1+c5fKW1+jhlZEM+8aWuQehE8UAAAAAAAAAAAAAAAEAAAAEdGVzdAAAABAAAAADZXZlAAAABWFsaWNlAAAAAGS0cnQAAAAAY8XhBAAAAEgAAAAab21lZ2EtcGVybUBjcnlwdG9ncmFwaHkuaW8AAAAAAAAAHmFscGhhLW9yZGVyaW5nQGNyeXB0b2dyYXBoeS5pbwAAAAAAAAAAAAAAAAAAAGgAAAATZWNkc2Etc2hhMi1uaXN0cDI1NgAAAAhuaXN0cDI1NgAAAEEET7TXKVMaHak8L5dyeoHUtWPY7ozrqZm82pkGh4nDDhV/ftU4eHq/D4FAVX0ETt1mvhZMWdcB8cKBkM75y96zUAAAAGMAAAATZWNkc2Etc2hhMi1uaXN0cDI1NgAAAEgAAAAgVN9wU6zFsYQbPdI4MeRXpPuWRBx+jJM8PWVWvP7QYysAAAAgWXE00j8SKuyJO4X/Jz9QIyLfkEmcXqoHkLVnPRoRZ10=
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-p256-non-lexical-extensions.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgTbZxg0yj7wdJdgLZWEmyQ27GJzA6hm0qmproBqnoGc0AAAAIbmlzdHAyNTYAAABBBKGL/tGE87lFxV86H/xeWCOc5EOwjojk5Mju0yM/z37Jgg1UJT0RYbZQdPEuFHlXrUsxNCgf7skyPFWoc784IoQAAAAAAAAAAAAAAAEAAAAEdGVzdAAAABAAAAADZXZlAAAABWFsaWNlAAAAAGS0cnQAAAAAY8XhBAAAAAAAAAAxAAAACnBlcm1pdC1wdHkAAAAAAAAAF3Blcm1pdC1hZ2VudC1mb3J3YXJkaW5nAAAAAAAAAAAAAABoAAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBOfVQv1ctoUn0QWVDCUViWOscuAi/ccYii4reJ9oUFMI0yJAM8HfLH8SQpGOQLzdP+Gr0YaLyDu/EFiZt9cMXwIAAABkAAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAABJAAAAIQCOQPb59BMqmZVlGvo/JYclnpWqIGiPB0c3Y1QZux/lXQAAACB0R5iKCzLmHo5A9wfsOv7oYLfDYGWSaA4WbBKJK3ETRw==
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-p384.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAghNd9jFDm9sueG2QuKV7fTRhKPKB0pprZqRP44dYXSZwAAAAIbmlzdHAyNTYAAABBBIdWgz67U4Fg01DzNm9UmX9EdFEqrfSyWj0ay2JLiCj8I+GHUoSTxOSrj0XkWvVdVpTAkmOX/cQrfJwRwDO71MUAAAAAAAAAAAAAAAEAAAAEdGVzdAAAAAAAAAAAZLRydAAAAABjxeEEAAAAAAAAAAAAAAAAAAAAiAAAABNlY2RzYS1zaGEyLW5pc3RwMzg0AAAACG5pc3RwMzg0AAAAYQSSpNYuZi/yHiolKGD8FG5oJH8aohYgCxp+u2U3sH7UKnaTOwQUU7hYdbl/YH2Ab+vCZ3ZHx/5naCgo4nTzotyegQITU3aeOS+Ivh8P4zsNc0PrnihFknh1DVLW9w09+8gAAACEAAAAE2VjZHNhLXNoYTItbmlzdHAzODQAAABpAAAAMQCtcUgAwYP8j657ThbfsTUZz1aiZZL0s1OfSkQKYJ49PJIEWDwZbgTohIG3D0j8PbkAAAAwT0ELlMrSu2sMcqRzbhcOK3KZgkZqdRW+5BHrc7TvYQQiYtOepqPjAC0LhRObcrep
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-p521.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAga9pP3iYGmmVlFMM97Lb2Nuj7LCi4fVeLN++/xsyHLOoAAAAIbmlzdHAyNTYAAABBBD42p7j3tjDcn1YoNjKgDgec+9Czi+6KNZM9EV51kpSq1ZZz1D5H+iqWByjmT5EtWCPq+2EgdprznvJgRMVejswAAAAAAAAAAAAAAAEAAAAEdGVzdAAAAAAAAAAAZLRydAAAAABjxeEEAAAAAAAAAAAAAAAAAAAArAAAABNlY2RzYS1zaGEyLW5pc3RwNTIxAAAACG5pc3RwNTIxAAAAhQQAP2p5Dgl+/P+tRZAsAoV3enPh2oCHheAWHjMLb1tlLPd9uZWH6vG/lb1SElA0KqMsRMNIjGfx2m1DGHuV6LirL/MBMWYTBhvrbMi+ltTd8cMiKaZKwAGcxJGbPtuRR/26qM8w8Aw04l5k4Ipy1l7dDjmO4QkI1rB2kAXHw0u/Wf0UBIsAAACnAAAAE2VjZHNhLXNoYTItbmlzdHA1MjEAAACMAAAAQgHbsKp+DSYdwqFDwk3/sF7vAy1rliZG8LYJqwAS6KvyncYgRVGGy2dP3KcBVnGn1DVwBXG4JNzHMZZolUgrK2nxiwAAAEIBhW3gjZ/7vA/Dy8azlubecyPfeLyegulmxvvJ6whmT/xQtkdSAx+JnLERN3cGRLToKA4TTmF0DG/dnfFOQygkMJE=
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-rsa-sha1.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgUvBup0dKwDazWqFmuY+CX0+O9SvgYe6XXwZdGSsCFoAAAAAIbmlzdHAyNTYAAABBBCRtIgvrukZwrNM62pkYACgJjoL0ViIzOZTvyuoaPRxbKTpVUQVeLt+H6kZDnrsloOzfH0P12DN3L4W6+rSyZnAAAAAAAAAAAAAAAAEAAAAEdGVzdAAAAAAAAAAAZLRydAAAAABjxeEEAAAAAAAAAAAAAAAAAAABFwAAAAdzc2gtcnNhAAAAAwEAAQAAAQEA5KypvcBCHtATD1GEgz8s2diLeb7efvSzeYHWmxA/gbuhEtT8grhqi9fi7ZU8Y4+5jL5Wk8fRMhiscle8EM7JeQb20Pxd3c67OfuIXK654q1wyQs60xd9LXEwdJguUsC3KsayZgM42vEg0Z0ZTv3P/i9ZnIdRp+3avIrMmIUrizm7oTIyu/u1QAO8fOLtULtkOLU47vs1h9vM9q3LJMn+s/DcrrHX8Wfto+9Cvbb+B99LUycP2rDNzebs8tMYYAVKwAxEYLbbXeTIhj649/QDyUtGbA5GvYZEB6XVDwD5EYtKDJtZv2lCq3eECklAlzP9j+VGBd6WThNaHZEPklZ/ewAAAQ8AAAAHc3NoLXJzYQAAAQBzQwOIXymvAyvyNhgTSPIi2FFKQdTcjz/2hgoLtroreN/5NMz2DkACChQo6qe5YMJHHsv5JVA4+gdVEMRxp1pL8RASPuSHHN9PSizDrNKZ2MT+woEE1nSFeKj9YzhSTMxvLti67W+FPnw7S39Bf7xrLkWbds+/rwoBeeYXAvwA/4DH0aplF1BUpC5dCFxhaPyVY6MgDC9J5nRxgbuATE40Hn1XwEl5EjKCMSyjnWU6Aqe3b5jJ5HAQFTe3Jmb4fBkivKxvwmbEHDfOLWk0ZMMGZMbHMUT7dSiy5Hjh1J5S8xMnbgPXuFer3q0+4FmdJwvqty+XHSTkwFP53Msw//Gz
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-rsa-sha256.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgEZ5t7LyaprT8g277kTvOMv/kgHECdpguBplHoRgAYpoAAAAIbmlzdHAyNTYAAABBBLQNIj2s2PZDa7LFPocsNN6ORF1ZsPmXqII23UPlpvFcUa8vDIccoO2shfUrFAMoudRiiyEHaLnpJCjIAvlM+0wAAAAAAAAAAAAAAAEAAAAEdGVzdAAAAAAAAAAAZLRydAAAAABjxeEEAAAAAAAAAAAAAAAAAAABFwAAAAdzc2gtcnNhAAAAAwEAAQAAAQEAnT7TBoEM1/ezloAEJ77uOOyRcx9K/PB/NPVVjTR6QgYirEGBkHOfxYBw7d2LlbdaZwiSOirwxj+rbox9+68w5XK/83P4YrxblxycCL6m/9R7y9k0B2TtAnq41Hf59CFZ3dI5WI3tCM2nUr/sUNRIjWelxA3EdTZ5eGhKia2B+dWVn+4YePcJXbasmPUdIA+7a+jdJ5znPiCUOA7bg3JjAOky9rKE/LiXJbpKl20D3RvXbIWosPGo+c93I3buGOFKGYNyWkYI4fljEZlVS5mZYGwG7xpdUq4ATK85lRB/RYU85gJMUdHA+2kqhXKBmMY1PPBoac2RF0zkDrQMbw1SmwAAARQAAAAMcnNhLXNoYTItMjU2AAABAIvIhSDBZwIL81IFD5dCOkT9UWy6PqHZo6xOIWXq/v2y0xZ7WLG4i5alCwxr9R8trBnrEjIrcA9XZVKawC11kaXxTxloLf5HnnEs6e49YewymgmwPMalLUqqJsAIDgdFN42g3UbeLF06a8PsHtcAui08/+xffgKNGV83+QpH+J5F8ykwWKmpc48NjulcTgUbvFmL1qOkt+Jq7ac/Hyrvu/KFx9YibCWSVLOSwL9wwQLBwq3Ham9ggnjBVj2rOfPwMBFGKaXvjb5LUlYMt2LjiPQJVVdAAKjvo/UjTWb5N21hQdtd3QEHViD+dYMkmLCCyo1U2Wfy5yfk05Hy3XqFLe8=
vectors/cryptography_vectors/asymmetric/OpenSSH/certs/p256-rsa-sha512.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgIOnlOEWH3RJ+U/Yr/pGEU2ig0h5qeLO9XgwIzocUxgUAAAAIbmlzdHAyNTYAAABBBAR7IuTauq/eQ9DgGVuv9qJ1jmI89k/9VIpKD/edK/mI9Rhz7RprmXC2Heopinwt9vyzvrNts7EwXasmIQcGbq4AAAAAAAAAAAAAAAEAAAAEdGVzdAAAAAAAAAAAZLRydAAAAABjxeEEAAAAAAAAAAAAAAAAAAABFwAAAAdzc2gtcnNhAAAAAwEAAQAAAQEAtsv6VhNt0q6UiBtzVOamVUpjEpLljN/r1yE9YkZdbEX9dmJHFMjr5uJxBT+PQ4w3lzdW0rnDUgX2HU8UkmzwOXCwj8USzs9Mxt9yL78hIVeZL2yRx4jYaWJsw58LzT48GyQGoW+/+FmCI+GBCMoiQ4ehTRhqREpwe0PM3iChc/9XBagjeXDmZir7rHvP3X3R04qCf+ybVv56mWIqWJAizfMMLASe6tnWvz5vvIVwJoCG6oZQeCMVXe4eBbq/gOLrUrnFtHtdpVAS15mE2tvpmG39wYc/fSmgJJC4owg62s1rCSYm+U8SWt9mkyK8BpVZvOcbf/6TXP81AnKqhL0nVwAAARQAAAAMcnNhLXNoYTItNTEyAAABAJpzmavIr0HwFiEX5wCDO+hvR7pKor2kWxSo1dYW+Hgk3DN/pyEc8GQ+jFFY1eiK1mOJ6MPIR/SfJbtIUM0OmfuLIYVGTl6bl+a/faTuvmzXgOWZ9DYtY1NYTWqLkIGgfXFwc3E4Hjamc+t9TD3aN35dYRfJFuMPZr4RmJCKEKlXBnrjFQfJQFCgzFmoFVp3jr+K5odM7oQ/JXmM3xZILeUHTNcC7xXljmweCbqfmRQtlkUheASkyPiV/tR7iz/e+hk17au+X2yymmL27SgK4CHSTcb4hGePUW1DQF7fk2AocEYr/UrXep3TXShBDQtYcJ0A7JaQaI+cEav6wN1EQZQ=
vectors/cryptography_vectors/asymmetric/OpenSSH/dsa-nopsw.key-cert.pub:ssh-dss-cert-v01@openssh.com AAAAHHNzaC1kc3MtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgT/srHasDLk3rB8bDZK7rP6miircgVvnVrTghYfkmZsEAAACBANGqIFBXxBjpr/l9S8UKzmSh4mBePW/YPp2VKoQEQS7xpED+wlBmZLHpfOiL2CWe/KtLT9KN/RjgnmJTUPSBiw3MGHFkUXKuDosTHBJ+oxxZYHDyl1wxw6NV+txaiG+oX81rYSfNNmBI5fdZtcDyKTDUitqTky8PuW40MtfZkqLvAAAAFQD3Hz77495ou42M1S9toNeG+bCNtwAAAIEAqWR4kR0VUOlkOYwbY5vwpfLyGup8TBkdHUHQc9aZmg6nD5Olj+hgTC4D2ZdIbQR4NVxCUEhoC27hl+ZlCdFDlxn5wFfxWfEABHolRDDlHTRTpxuINJhcOf3ewtb4Pbt/WOc1KGzLPjcXrouuxdZMuagW0wpfD5gEvlwvsx+To5wAAACAeqA6atklGY2JEkBj/TKU4H2lxLQ59hKPcxciBuRLSpG9H4uW/VCkUKrKTNMQU42ixlvvklvcW4irDVZ1kE4zxkXl0gF0cCxDTU2hNmIclFCUXAGoxngMKze3EO4zUZmYNphYhF3NWeurxRKoFKja3UG9dtlcUtyYewjKHGH0K6wAAAAAAAAAAQAAAAEAAAAEbmFtZQAAAAAAAAAASz3OqAAAAAEFdF0oAAAAAAAAAIIAAAAVcGVybWl0LVgxMS1mb3J3YXJkaW5nAAAAAAAAABdwZXJtaXQtYWdlbnQtZm9yd2FyZGluZwAAAAAAAAAWcGVybWl0LXBvcnQtZm9yd2FyZGluZwAAAAAAAAAKcGVybWl0LXB0eQAAAAAAAAAOcGVybWl0LXVzZXItcmMAAAAAAAAAAAAAAbIAAAAHc3NoLWRzcwAAAIEA0aogUFfEGOmv+X1LxQrOZKHiYF49b9g+nZUqhARBLvGkQP7CUGZksel86IvYJZ78q0tP0o39GOCeYlNQ9IGLDcwYcWRRcq4OixMcEn6jHFlgcPKXXDHDo1X63FqIb6hfzWthJ802YEjl91m1wPIpMNSK2pOTLw+5bjQy19mSou8AAAAVAPcfPvvj3mi7jYzVL22g14b5sI23AAAAgQCpZHiRHRVQ6WQ5jBtjm/Cl8vIa6nxMGR0dQdBz1pmaDqcPk6WP6GBMLgPZl0htBHg1XEJQSGgLbuGX5mUJ0UOXGfnAV/FZ8QAEeiVEMOUdNFOnG4g0mFw5/d7C1vg9u39Y5zUobMs+Nxeui67F1ky5qBbTCl8PmAS+XC+zH5OjnAAAAIB6oDpq2SUZjYkSQGP9MpTgfaXEtDn2Eo9zFyIG5EtKkb0fi5b9UKRQqspM0xBTjaLGW++SW9xbiKsNVnWQTjPGReXSAXRwLENNTaE2YhyUUJRcAajGeAwrN7cQ7jNRmZg2mFiEXc1Z66vFEqgUqNrdQb122VxS3Jh7CMocYfQrrAAAADcAAAAHc3NoLWRzcwAAAChGJhskBvKjziUaQuE3Kd3A+3WqQduqROikisQyJqTEmD9CqCrV2tuQ dsa-nopsw.key
vectors/cryptography_vectors/asymmetric/OpenSSH/ecdsa-nopsw.key-cert.pub:ecdsa-sha2-nistp256-cert-v01@openssh.com AAAAKGVjZHNhLXNoYTItbmlzdHAyNTYtY2VydC12MDFAb3BlbnNzaC5jb20AAAAg7ohu2h3ZSjGNlNGn5wmDHarKBT8q+6Yl23V+s3MNjzEAAAAIbmlzdHAyNTYAAABBBCZWRs4GYIHGJpyXuqvfFGWN49dnJRkZJLDkFrHf6mNHhIMI3vtrLfCZwxPSfnCYWK6YofssZ1FYA6TkVJq8Xi8AAAAAAAAAAAAAAAIAAAAEbmFtZQAAABYAAAAHZG9tYWluMQAAAAdkb21haW4yAAAAAAAAAAD//////////wAAAAAAAAAAAAAAAAAAAGgAAAATZWNkc2Etc2hhMi1uaXN0cDI1NgAAAAhuaXN0cDI1NgAAAEEEJlZGzgZggcYmnJe6q98UZY3j12clGRkksOQWsd/qY0eEgwje+2st8JnDE9J+cJhYrpih+yxnUVgDpORUmrxeLwAAAGMAAAATZWNkc2Etc2hhMi1uaXN0cDI1NgAAAEgAAAAhAOSy1UooQNXaxG4cbZTQVnC7uSJlLuk4w9Z5XvbfzzmSAAAAH2QDu0n+WcXzDEsXeaH6IV0drX99PYLeiabxYeUha/o= ecdsa-nopsw.key
vectors/cryptography_vectors/asymmetric/OpenSSH/ed25519-nopsw.key-cert.pub:ssh-ed25519-cert-v01@openssh.com AAAAIHNzaC1lZDI1NTE5LWNlcnQtdjAxQG9wZW5zc2guY29tAAAAIJjE7jTtIuJ68m5zBEtn0bCWKUXcMdAOau0hO3FTMpeAAAAAIN1mDO2AAUULtPk+J+tTL+Qy7Q+fCrVq7e9K0od7sUUwAAAAAAAAAAAAAAABAAAABG5hbWUAAAAAAAAAAAAAAAD//////////wAAAAAAAABkAAAAFXBlcm1pdC1YMTEtZm9yd2FyZGluZwAAAAAAAAAXcGVybWl0LWFnZW50LWZvcndhcmRpbmcAAAAAAAAACnBlcm1pdC1wdHkAAAAAAAAADnBlcm1pdC11c2VyLXJjAAAAAAAAAAAAAAAzAAAAC3NzaC1lZDI1NTE5AAAAIN1mDO2AAUULtPk+J+tTL+Qy7Q+fCrVq7e9K0od7sUUwAAAAUwAAAAtzc2gtZWQyNTUxOQAAAEAqewM0LFjD4qQUdGNmF/W512ogcjh5xDchw9h2GjhFstttkQVfEOATyafZ5/vWGegMjSnGWHHTxv1A5bqzA+UE ed25519-nopsw.key
vectors/cryptography_vectors/asymmetric/OpenSSH/rsa-nopsw.key-cert.pub:ssh-rsa-cert-v01@openssh.com AAAAHHNzaC1yc2EtY2VydC12MDFAb3BlbnNzaC5jb20AAAAgtL8iJdlD7w2obBZdlxyqu06uLR78fvDASlnR3RRk/eMAAAADAQABAAABAQCrZB1KbbZASyb4a2kDA3iqTR7EMjHDFRvyMnossujX0olH3JJDD50COn1YhS6ZczY49iLSqq03okoywOtVJHztGgrtXZbiUdiWNkE4MZgDxt+mpou3kcy+QmmyQwQGomRm3A7wb8ndAM4UHKIY7F54vj/a+czlfA399/pIoAHF9TrSAOaWh8WVmoIP17FY7nZ2DkFD+A3moksEoFmYLeNacCStuW98/HB+n1YLeAysUggvK0Ja9G71goz6dBmiqjTIJvMPLsZErldkmUd/D6NXcxYxJdLsZKKs6ZO2bLKAK2SmEDdjwcBCWJuF2D7mNkNZF0NchTa44ROIdzyq/EzzAAAAAAAAAAIAAAABAAAABG5hbWUAAAASAAAABXVzZXIxAAAABXVzZXIyAAAAAAAAAAD//////////wAAAAAAAACCAAAAFXBlcm1pdC1YMTEtZm9yd2FyZGluZwAAAAAAAAAXcGVybWl0LWFnZW50LWZvcndhcmRpbmcAAAAAAAAAFnBlcm1pdC1wb3J0LWZvcndhcmRpbmcAAAAAAAAACnBlcm1pdC1wdHkAAAAAAAAADnBlcm1pdC11c2VyLXJjAAAAAAAAAAAAAAEXAAAAB3NzaC1yc2EAAAADAQABAAABAQCrZB1KbbZASyb4a2kDA3iqTR7EMjHDFRvyMnossujX0olH3JJDD50COn1YhS6ZczY49iLSqq03okoywOtVJHztGgrtXZbiUdiWNkE4MZgDxt+mpou3kcy+QmmyQwQGomRm3A7wb8ndAM4UHKIY7F54vj/a+czlfA399/pIoAHF9TrSAOaWh8WVmoIP17FY7nZ2DkFD+A3moksEoFmYLeNacCStuW98/HB+n1YLeAysUggvK0Ja9G71goz6dBmiqjTIJvMPLsZErldkmUd/D6NXcxYxJdLsZKKs6ZO2bLKAK2SmEDdjwcBCWJuF2D7mNkNZF0NchTa44ROIdzyq/EzzAAABFAAAAAxyc2Etc2hhMi01MTIAAAEAC13gJgpzMrO3fGM7lNpLLBDUa6GxB4Lr/RSHLxG5v9+Ym7z5aIYmneYifbNQ9qIPDxpuBQvN97NgV4ImSMjfIaaQi/PxyHKC+mxMXAonT5c5d+zhZHQ95/uxIt6iba1ej3MRVYw+biQlnIHhOUeJNV3W8UOry0XQlJzjTBR2/1Y+Xk89NbZ3F6QQ9UZpT/FvbERghElBauTz1UhtSlMCrn66tEOGxreVHraoM92pJaygginwb5iVU0SPWhWG3Qyh8P6uKzqSKIr+6BfaiIoWJXOLLEtSfmuK8dTuE8fNeU4PQlkJBqfcWbpD934QsP1dXWhan5Y4iFMK4IjcPNu7iA== rsa-nopsw.key
```